lib/segment/src/telemetry.rs

```diff
@@ -84,7 +84,21 @@ impl Anonymize for SegmentInfo {
             ram_usage_bytes: self.ram_usage_bytes.anonymize(),
             disk_usage_bytes: self.disk_usage_bytes.anonymize(),
             is_appendable: self.is_appendable,
-            index_schema: self.index_schema.clone(),
+            index_schema: self
+                .index_schema
+                .iter()
+                .map(|(k, v)| (k.anonymize(), v.anonymize()))
```

Comment:
Key is a string. Anonymizing of the string is a hash value of string. Hash values may be equal between different strings. We have to discuss ways of avoiding conflicts. I propose to add some special symbol to the anonymized key value if map already has same anonymized key

Reply:
that's a 64 bit hash. probability of the collision is sub atomic. And in the worst case we will just lose the record in statistics

Reply:
I checked that there is no panic while names collision. I asked about conflict because of panic. No panic - no problem =)